### PR TITLE
Added population views.

### PR DESCRIPTION
**Pull request by rasmi**

### CODE REVIEW

File: recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+# =============================================================================
+"""Total population, admissions, releases by race, gender and day-fips."""
+# pylint: disable=line-too-long
+
+from recidiviz.calculator.bq import export_config
+from recidiviz.calculator.bq.views import bqview
+
+from recidiviz.persistence.database.schema import Booking
```
Comment (ohinds) [inline recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py]: one line?

File: recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  JOIN Days
+  ON day = BookingDates.release_date
+  GROUP BY day, fips, race, gender
+),
+
+PersonCountTable AS (
+  SELECT day, fips, race, gender, COUNT(DISTINCT(booking_id)) AS person_count
+  FROM BookingDates
+  JOIN Days
+  ON day BETWEEN BookingDates.admission_date AND COALESCE(BookingDates.release_date, CURRENT_DATE('America/New_York'))
```
Comment (ohinds) [inline recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py]: Is this robust to the 'scrape across midnight' issue?

File: recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py (hunk 1 above)
Comment (rasmi) [reply recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py]: I like importing on different lines, makes diffs easier.

File: recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py (hunk 2 above)
Comment (rasmi) [reply recidiviz/calculator/bq/views/population/population_admissions_releases_race_gender.py]: No, but that was resolved by setting all releases times to be set on scraper_start -- @terinpw?

**Pull request merged**